zencode/Keypair-Creation-Client-Side.keys

```diff
@@ -1,11 +1,11 @@
 {
 	"userChallenges": {
-		"whereMyParentsMet": "Paris",
-		"myFirstPet": "ScoobyDoo",
-		"myHomeTown": "Amsterdam",
-		"nameOfFirstTeacher": "null",
-		"surnameOfMotherBeforeWedding": "null"
+		"question1": "Paris",
+		"question2": "ScoobyDoo",
+		"question3": "Amsterdam",
+		"question4": "null",
+		"question5": "null"
 	},
-	"username": "JohnDoe",
+	"user": "JohnDoe",

```

Comment:
Typo? Seems that is inconsistent with the contract looking for `username`

Reply:
.keys files are not used at all. they're there just as an example but it should be 'username'

Reply:
keys are not used at all.. they're there just as an example

Reply:
Changed to username